pac4j-core/src/main/java/org/pac4j/core/io/InputStreamSource.java

```diff
@@ -0,0 +1,24 @@
+package org.pac4j.core.io;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Simple interface for objects that are sources for an {@link InputStream}.
+ *
+ * <p>This is the base interface for @link Resource} interface.
+ *
+ * @see java.io.InputStream
+ * @see Resource
+ */

```

Comment:
After the hard work you are doing, I guess you don't want to forget the `author` annotation as well. I think I saw the problem in several other files.